lib/instrumenter/collectors-gz/zpool.js

```diff
@@ -70,6 +70,28 @@ function getZpoolKstats(opts, callback) {
     }, callback);
 };
 
+/*
+ * If present, chop off the trailing '%' sign.
+ * This is a bug in the 'zpool' command. Currently when using the '-p'
+ * flag the fragmentation field _will_ include a % sign, but the
+ * capacity field will _not_ contain a % sign.
+ */
+function zpoolFragEnc(x) {
+    return x.replace(/%$/g, '');
+}
+
+/*
+ * Encode zpool health status into a numberic value.
+ */
+function zpoolHealthEnc(health) {
+    var zpoolStatus = {
```

Comment:
We could make this an array and do something like
```
return zpoolStatus.indexOf(health);
```
That should behave the same way as this code and be a little more concise.

Reply:
Nice idea - The code has been updated to use `indexOf()`.